integration/tests/basic_functionality/negative/neg_deploy_with_missing_r10k_yaml.rb

```diff
@@ -0,0 +1,31 @@
+require 'git_utils'
+require 'r10k_utils'
+require 'master_manipulator'
+test_name 'CODEMGMT-84 - C59270 - Attempt to Deploy with Missing r10k Configuration File'
+
+#Init
+env_path = on(master, puppet('config print environmentpath')).stdout.rstrip
+git_repo_path = '/git_repos'
+git_control_remote = File.join(git_repo_path, 'environments.git')
+
+r10k_config_path = '/etc/r10k.yaml'
+r10k_config_bak_path = "#{r10k_config_path}.bak"
+
+#Verification
+error_message_regex = /Error while running.*R10K\:\:Deployment.*No configuration file/
+
+#Teardown
+teardown do
+  step 'Restore Original "r10k" Config'
+  on(master, "cp #{r10k_config_bak_path} #{r10k_config_path}")
```

Comment:
You should use "mv" instead so you don't leave behind the old backup file which might cause confusion when debugging other tests.